src/util/long.js

```diff
@@ -0,0 +1,27 @@
+const BigNumber = require('bignumber.js')
+const { Reader, Writer } = require('oer-utils')
+
+function longToBuffer (_n) {
+  const n = new BigNumber(_n)
+  const high = n.div(0x100000000)
+  const low = n.mod(0x100000000)
+
+  const writer = new Writer()
+  writer.writeUInt64([ high, low ])
+  return writer.getBuffer()
+}
+
+function bufferToLong (buffer) {
+  const reader = new Reader(buffer)
+  const [ high, low ] = reader.readUInt64()
+
+  const longHigh = new BigNumber(high)
+  return longHigh
+    .mul(0x100000000)
+    .add(low)
+}
+
+module.exports = {
+  longToBuffer,
+  bufferToLong
+}
```

Comment:
Where are these functions used?

Reply:
oh, looks like those are leftover from when I began implementing the OER balance sub-protocol. I'll remove them from this version, because the balance sub-protocol can't be implemented until it uses signed integers.

Reply:
It does now: https://github.com/interledger/interledger/wiki/Interledger-over-BTP#changes-to-balance-sub-protocol